### PR TITLE
Debt team - 95944 Add retry for VBA submit

### PR DESCRIPTION
**Pull request by digitaldrk**
We are getting Time Out errors when submitting to VBA. This adds in retrying with hopes that we can get submissions flowing through when their system can handle traffic again.

## Summary

- *This work is behind a feature toggle (flipper): YES/NO*: NO
- *(Summarize the changes that have been made to the platform)*: We add retries, just 5. We also spread the retires out so we don't keep hammering their API since there are timeouts happening.
- *(If bug, how to reproduce)*: N/A
- *(What is the solution, why is this the solution?)*: The idea here is to get the form submitted without the need for manual intervention. Since the API is timing out, I thought it would be helpful to wait sometime to try again rather than the default 'exponential backoff' that Sidekiq does by resubmitting in quick succession at first.
- *(Which team do you work for, does your team own the maintenance of this component?)*: I work on the Debt Resolution team. You can find my on the OCTO slack on the #debt-resolution channel.
- *(If introducing a flipper, what is the success criteria being targeted?)*: N/A

## Related issue(s)

- *Link to ticket created in va.gov-team repo OR screenshot of Jira ticket if your team uses Jira*: https://app.zenhub.com/workspaces/vsa---debt-607736a6c8b7e2001084e3ab/issues/gh/department-of-veterans-affairs/va.gov-team/95904
- *Link to previous change of the code/bug (if applicable)*
- *Link to epic if not included in ticket*

## Testing done

- [x] *New code is covered by unit tests*
- *Describe what the old behavior was prior to the change*: The old behavior was to try once and then run the exhaustion block. Now we retry up to 5 times with an hour between attempts and then run the exhaustion block.
- *Describe the steps required to verify your changes are working as expected. Exclusively stating 'Specs run' is NOT acceptable as appropriate testing*: It's tough to test this manually. I'm open to ideas.
- *If this work is behind a flipper:*: N/A
  - *Tests need to be written for both the flipper on and flipper off scenarios. [Docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Backendexample).*
  - *What is the testing plan for rolling out the feature?*

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: modules/debts_api/app/sidekiq/debts_api/v0/form5655/vba_submission_job.rb [hunk 1]
```diff
@@ -8,21 +8,24 @@ class V0::Form5655::VBASubmissionJob
     include SentryLogging
     STATS_KEY = 'api.vba_submission'
 
-    sidekiq_options retry: false
+    sidekiq_options retry: 5, retry_interval: ->(_) { 1.hour.to_i }
```
Comment (digitaldrk) [inline modules/debts_api/app/sidekiq/debts_api/v0/form5655/vba_submission_job.rb:11]: I'm interested to hear if you(the reviewer) have better ideas than what I've done here :).

**Review by penny-lischer — APPROVED** [review]
LGTM

File: modules/debts_api/app/sidekiq/debts_api/v0/form5655/vba_submission_job.rb (hunk 1 above)
Comment (rmtolmach) [reply modules/debts_api/app/sidekiq/debts_api/v0/form5655/vba_submission_job.rb:11]: @digitaldrk Would `sidekiq_retry_in` be better? I see it used a few times in vets-api.
This is the most documentation I could find: https://github.com/sidekiq/sidekiq/blob/main/lib/sidekiq/job_retry.rb#L203-L205

**Review by rmtolmach — APPROVED** [review]
LGTM!

**Pull request merged**